samples/chatbot-web-search/src/main/resources/META-INF/resources/components/demo-chat.js

```diff
@@ -12,6 +12,7 @@ export class DemoChat extends LitElement {
                 right: false,
                 sender: {name: 'Bob', id: '007'}
             });
+            chatBot.hideAllLoading();
```

Comment:
I would hide first and display after (just move this before the addMessage)

Reply:
Ok done